### ISSUE TITLE
Unable to change key name

### ISSUE THREAD

**Issue by lemongrabber** [ConfigHubPub/Docs#1]
When attempting to change a key name to another name that has already been referenced in a file somewhere (even if it doesn't have any values), after clicking the Save Key, button you get a "Unable to process request" error. Screenshots of the UI are attached, and the message in the confighub logs is:
```
2017-05-22 17:45:35,673 ERROR ----------------- Un-handled exception start --------------------
2017-05-22 17:45:35,674 ERROR java.util.ConcurrentModificationException
2017-05-22 17:45:35,674 ERROR ----------------- Un-handled exception end   ------------------
```
<img width="1186" alt="screen shot 2017-05-22 at 2 00 16 pm" src="https://cloud.githubusercontent.com/assets/6097892/26324261/b0aaf65c-3ef7-11e7-87e9-cbd500cb8486.png">
<img width="1162" alt="screen shot 2017-05-22 at 1 59 10 pm" src="https://cloud.githubusercontent.com/assets/6097892/26324262/b0ae9334-3ef7-11e7-89fb-e17ccb66f1e5.png">

**Comment by edoarsla** [comment]
We're looking into this...

**Comment by edoarsla** [comment]
For future reference, please use the this repo for issues:

https://github.com/ConfigHubPub/ConfigHubPlatform/issues

**Comment by edoarsla** [comment]
Hi.  We just released a fix for this - v1.3.5.  

https://github.com/ConfigHubPub/ConfigHubPlatform/releases/tag/v1.3.5

**Comment by lemongrabber** [comment]
Awesome, thanks for the quick turn-around!
